tests/testLinearizeLookupTable.py

```diff
@@ -0,0 +1,232 @@
+import unittest
+import cPickle
+
+import numpy as np
+
+import lsst.utils.tests
+import lsst.pex.logging as pexLog
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.table as afwTable
+import lsst.afw.cameraGeom as cameraGeom
+from lsst.ip.isr import applyLookupTable, LinearizeLookupTable
+
+
+def refLinearize(image, detector, table):
+    """!Reference implementation of non-linearization correction
```

Comment:
This is not a reference implementation, it is a copy of the actual implementation.  A reference implementation must have been tested and validated; this has no pedigree.